genet/core.py

```diff
@@ -2023,6 +2037,14 @@ def generate_validation_report(self, modes_for_strong_connectivity=None, link_me
             graph_connectivity[mode] = self.check_connectivity_for_mode(mode)
         report['graph'] = {'graph_connectivity': graph_connectivity}
 
+        has_isolated_nodes = self.has_isolated_nodes()
+        report['graph']['isolated_nodes'] = {
+            'has_isolated_nodes': str(has_isolated_nodes),
```

Comment:
I would maybe get rid of this line and then just keep an `isolated_nodes` key for simplicity, so it just shows `None` if there aren't any? Or maybe at least make it a number of isolated nodes, then have a separate list of them under `nodes`?

Reply:
good idea, i'll make it so:
```python
report['graph']['isolated_nodes'] = {
    "number_of_nodes: 3,
    "nodes" : [A, B, C]
}
```
or 
```python
report['graph']['isolated_nodes'] = {
    "number_of_nodes: 0,
    "nodes" : []
}
```
if there aren't any 👍